Neos.Fusion/Classes/Core/FusionSourceCodeCollection.php

```diff
@@ -52,6 +52,9 @@ public static function tryFromFilePath(string $filePath): self
         return self::fromFilePath($filePath);
     }
 
+    /**
+     * @deprecated with Neos 9, remove me :)
```

Comment:
why, what will it be replaced with?

Reply:
this is not api anymore with Neos 9.0 and you can make it yourself ... there is just no need for this as we dont use it for the normal fusion view neither. The source code collection should not deal with this logic and be flow package aware.